api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/Parameter.scala

```diff
@@ -14,19 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package ai.h2o.sparkling.ml.algos
 
-import ai.h2o.sparkling.ml.params.H2OGLMParams
-import ai.h2o.sparkling.ml.utils.H2OParamsReadable
-import hex.glm.GLMModel.GLMParameters
-import org.apache.spark.ml.util.Identifiable
+package ai.h2o.sparkling.api.generation.common
 
-/**
-  * H2O GLM algorithm exposed via Spark ML pipelines.
-  */
-class H2OGLM(override val uid: String) extends H2OSupervisedAlgorithm[GLMParameters] with H2OGLMParams {
+case class Parameter(swName: String, h2oName: String, defaultValue: AnyRef, dataType: DataType, comment: String)
```

Comment:
Do we store the mapping also in generated classes? Especially `swName: String, h2oName: String` will help a lot with GridSearch ( allowing to pass sw param names)